pkgs/development/python-modules/django-registration/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  confusable-homoglyphs,
+  coverage,
+  django,
+  fetchFromGitHub,
+  pdm-backend,
+  pythonAtLeast,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "django-registration";
+  version = "5.1.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9" || pythonAtLeast "3.13";
```

Comment:
Upstream claims to support 3.13 https://github.com/ubernostrum/django-registration/blob/5.1.0/pyproject.toml#L25

Reply:
Perhaps you wanted `pythonAtLeast "3.14"`? (`pythonOlder` is <, `pythonAtLeast` is >=)